### DISCUSSION TITLE
Allow creating a branch in a fork even with write access to the repository

### DISCUSSION THREAD

**Discussion by xmo-odoo** [community/community#57205] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

This is an issue for both codespaces (which I assume is github.dev) and the "basic" github code editor: when the user has write access to a repository (because they're a maintainer or the administrator for instance) the only options are:

- direct commit
- create a branch in the same repository

Both can be *useful* for emergency fixes, but neither is *desirable* if the normal workflow is to perform changes in a fork (either organisational or personal) then submit a PR.

Sadly there seems to be no way whatsoever to do that via the web.

**Comment by dmilburn** [comment, marked as answer]
hi @xmo-odoo, thanks for your feedback. i've submitted internal issues for your feature request but can't guarantee when they may be prioritized. 

I did want to call out, if a fork of a repo already exists, you can navigate directly to the fork and edit files from there.

**Reply by xmo-odoo** [reply]
@dmilburn aye, but it is quite frustrating not remembering (because it's not an issue on all repositories so you might not think of it upfront), doing the edits, then on commit remembering that you need to go over to the fork and having to move the edits over.

Thanks for making a note of it.
